docs/src/mixed_CPU_GPU.md

```diff
@@ -0,0 +1,45 @@
+# Mixed CPU and GPU computations
+
+If GPU memory is a limiting factor for your computation, it may be preferable to carry out particle operations on the CPU rather than on the GPU.
+This involves basically 4 steps:
+
+1) *At the top of the script*. The JustPIC backend must be set to CPU, in contrast with other employed packages (e.g. ParallelStencil):
+```julia
+const backend = JustPIC.CPUBackend 
+```
+
+2) *At memory allocation stage*. A copy of relevant CPU arrays must be allocated on the GPU memory. For example, phase ratios on mesh vertices:
+```julia
+phv_GPU = @zeros(Nc.x+1, Nc.y+1, Nc.z+1, celldims=(N_phases))
```

Comment:
```suggestion
phv_GPU = @zeros(nx+1, ny+1, nz+1, celldims=(number_of_phases))
```

Perhaps the variable names are more obvious in this way.

Reply:
perfect!